cores/nRF5/wiring.c

```diff
@@ -143,3 +143,27 @@ void systemOff(uint32_t pin, uint8_t wake_logic)
     NRF_POWER->SYSTEMOFF = 1;
   }
 }
+
+
+float readCPUTemperature()
```

Comment:
float is perfect, it is indeed the right choice for the function. Though can you add void as its explicit argument

```C
float readCPUTemperature(void)
```